docs/api_call.md

```diff
@@ -1,1352 +1,1154 @@
+## API Documentation
```

Comment:
Parts of this file is marked dark red (not diff red) which indicates potential Markdown formatting errors. Please consider those warnings to improve readability

Reply:
In lot of curl calls `None` is being used which is probably not correct as None is Python specific and passing it in curl would mean our APIs would read it as `'None'`

Reply:
One additional suggestion is to group all v2 detectors above v1 detectors . Makes it easier to understand what is on v2 and what is on v1